### ISSUE TITLE
Sharing of .gitignore

### ISSUE THREAD

**Issue by fulkast** [RPLtemp/traxers#1]
I have some local non-catkin files that I ignore and didn't want to pollute the .gitignore file in the repo. Hence I thought it best to just ignore .gitignore itself, which I think has been recommended as good practice.

**Comment by pvechersky** [comment]
Yeah, that sounds like a good practice! I'll just keep a local .gitignore in my work branch then and when the time comes to merge the code into master I just won't include the .gitignore in the merge.
